test/integration/reconnect/postgresContainer.js

```diff
@@ -0,0 +1,92 @@
+const Promise = require('bluebird');
+const _        = require('lodash');
+
+function PostgresContainer(docker, options) {
+  var name     = _.get(options, 'name', 'knex-postgres');
+  var image    = _.get(options, 'image', 'postgres:9.6');
+  var username = _.get(options, 'username', 'postgres');
+  var password = _.get(options, 'password', '');
+  var hostPort = _.get(options, 'hostPort', 5432);
```

Comment:
No. I mean that usually CI machine (and pretty much every developers machine) is already running the default postgres server in that port. So if you are trying to use the same port for postgres which is ran in docker, starting the server should fail.